docsite/rst/roadmap/ROADMAP_2_4.rst

```diff
@@ -6,6 +6,9 @@ Ansible by Red Hat, Core 2.4
 This is meant to be a living document, and is by no means **FINAL** until
 stated otherwise in the document.
 
+- **Python 2.4 support discontinuation**
+  - Python 2.4 will no longer be supported. Going forward, Python 2.6+ will be supported.
```

Comment:
bikeshedding:
"Python 2.4 will no longer be supported on target machines (controller already required >=2.6), from now on Ansible will require >=Python2.6"